services/ui-src/src/components/app/App.tsx

```diff
@@ -1,8 +1,17 @@
-import { Routes, Route } from "react-router-dom";
-import { LoginCognito, LoginIDM, PostLogoutRedirect } from "components";
 import { useStore } from "utils";
+import { Routes, Route } from "react-router-dom";
+import {
+  AppRoutes,
+  Error,
+  LoginCognito,
```

Comment:
Do we need the LoginCognito and LoginIDM for this setup yet?

Reply:
i believe so, it was added in by Brendan a few weeks ago because we're using it on lines 56-58!